connector/src/main/java/org/jboss/as/connector/deployers/ra/processors/CachedConnectionManagerSetupProcessor.java

```diff
@@ -90,6 +92,10 @@ private CachedConnectionManagerSetupAction(final ServiceName serviceName) {
         @Override
         public void setup(final Map<String, Object> properties) {
             try {
+                final CachedConnectionManager noTxCcm = noTxCcmValue.getOptionalValue();
```

Comment:
This looks like it will use both the TX and not TX version every time. This does not look right?